app/models/event.rb

```
@@ -0,0 +1,23 @@
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: "User", foreign_key: :creator
```

RuboCop (1.75.5)

[convention] Specify an `:inverse_of` option.

(Rails/InverseOf)